pkg/apis/datadoghq/v1alpha1/watermarkpodautoscaler_default.go

```diff
@@ -5,10 +5,14 @@
 
 package v1alpha1
 
-import "fmt"
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+)
 
 const (
-	defaultTolerance                       = 0.1
+	defaultTolerance                       = 100
```

Comment:
what is the unit?

Reply:
could you add a comment about this value? what should be the min-max for a tolerance value

Reply:
it's to be used later on as milliquantity, so we can get 10%